introduction_to_applying_machine_learning/identify_key_insights_from_textual_document/document_relationship_extraction.ipynb

```diff
@@ -0,0 +1,710 @@
+{
```

Comment:
<p>Desipte should be Desipte</p><p> </p><p>Use present tense - will be downloaded  should be  are downloaded </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>